### PR TITLE
docs: add eip4844 info to the on-chain data page

### PR DESCRIPTION
**Pull request by ArielElp**
### Description of the Changes

The on-chain data page was not updated since v0.13.1, we add a section that contains some information about blobs

### PR Preview URL

https://starknet-io.github.io/starknet-docs/pr-1247/documentation/architecture_and_concepts/Network_Architecture/on-chain-data/

### Check List

- [x] Changes have been done against main branch, and PR does not conflict
- [x] PR title follows the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1247)

### CODE REVIEW

**Review by stoobie — APPROVED** [review]
@ArielElp I made several edits inline. As long as you don't have an issue with my edits, LGTM.

File: components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/on-chain-data.adoc [hunk 1]
```diff
@@ -4,18 +4,30 @@
 [id="introduction"]
 == Introduction
 
-In the current stage of the Alpha, Starknet operates in Validity Rollup mode, also referred to as ZK-Rollup mode. So when the onchain state update is accepted, the state diff between the previous and new state is sent as calldata to Ethereum.
 
-This data allows anyone that observes Ethereum to reconstruct the current state of Starknet.
+Starknet operates as a Validity Rollup, so when consolidating a set of Layer 2 changes on Layer 1, it updates the latest proven Layer 2 state by publishing the state diffs, that is, the differences between the previous L2 state and the new L2 state, on Ethereum.
```
Comment (stoobie) [inline components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/on-chain-data.adoc:8]: @arielelp, How's this:

```suggestion
Starknet is a Validity Rollup, so after consolidating and proving a set of Layer 2 changes, it updates the latest proven L2 state by publishing the state diffs, that is, the differences between the previous L2 state and the new L2 state, on Ethereum.
```

**Review by ArielElp — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1247)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @stoobie)

___
*[`components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/on-chain-data.adoc` line 8 at r2](https://reviewable.io/reviews/starknet-io/starknet-docs/1247#gh-1590330196:-NxC9oKU8KMiPj-nVoJh:bp2ai61) ([raw file](https://github.com/starknet-io/starknet-docs/blob/167931ad333b5d24b4385059f732cdc81dd7642a/components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/on-chain-data.adoc#L8)):*
<details><summary><i>Previously, stoobie (Steve Goodman) wrote…</i></summary><blockquote>

@arielelp, How's this:

```suggestion
Starknet is a Validity Rollup, which means that after consolidating and proving a set of Layer 2 changes, it updates the latest proven L2 state by publishing the state diffs, that is, the differences between the previous L2 state and the new L2 state, on Ethereum.
```
</blockquote></details>


```
Starknet is a Validity Rollup, which means that after consolidating and proving a set of Layer 2 changes, it updates on L1 the latest proven L2 state. Alongside the proof, the state-diff between the old and new state are published on Ethereum.
```

**Review by stoobie — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1247)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @ArielElp)

___
*[`components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/on-chain-data.adoc` line 8 at r2](https://reviewable.io/reviews/starknet-io/starknet-docs/1247#gh-1590330196:-NxCPjxXDfUOCJ8RXLsq:b8a9qd4) ([raw file](https://github.com/starknet-io/starknet-docs/blob/167931ad333b5d24b4385059f732cdc81dd7642a/components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/on-chain-data.adoc#L8)):*
<details><summary><i>Previously, ArielElp wrote…</i></summary><blockquote>

```
Starknet is a Validity Rollup, which means that after consolidating and proving a set of Layer 2 changes, it updates on L1 the latest proven L2 state. Alongside the proof, the state-diff between the old and new state are published on Ethereum.
```
</blockquote></details>

How about this:

`Alongside the proof, it publishes the state diff on Ethereum. The state diff is the difference between the previous and new states.`

**Review by stoobie — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1247)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @ArielElp)

___
*[`components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/on-chain-data.adoc` line 8 at r2](https://reviewable.io/reviews/starknet-io/starknet-docs/1247#-NxCgDDp42MnW0NmuDiH:-NxCgDDp42MnW0NmuDiI:b9ux598) ([raw file](https://github.com/starknet-io/starknet-docs/blob/167931ad333b5d24b4385059f732cdc81dd7642a/components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/on-chain-data.adoc#L8)):*
> ```AsciiDoc
> 
> 
> Starknet operates as a Validity Rollup, so when consolidating a set of Layer 2 changes on Layer 1, it updates the latest proven Layer 2 state by publishing the state diffs, that is, the differences between the previous L2 state and the new L2 state, on Ethereum.
> ```

After discussing with @ArielElp, we'll go with:



_Suggestion:_
```AsciiDoc
Starknet is a Validity Rollup, which means that after consolidating and proving a set of Layer 2 changes, it updates on L1 the latest proven L2 state. Alongside the proof, it publishes the state diff on Ethereum. The state diff is the difference between the previous and new states.
```

File: components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/on-chain-data.adoc [hunk 2]
```diff
@@ -4,18 +4,30 @@
 [id="introduction"]
 == Introduction
 
-In the current stage of the Alpha, Starknet operates in Validity Rollup mode, also referred to as ZK-Rollup mode. So when the onchain state update is accepted, the state diff between the previous and new state is sent as calldata to Ethereum.
 
-This data allows anyone that observes Ethereum to reconstruct the current state of Starknet.
+Starknet is a Validity Rollup, which means that after consolidating and proving a set of Layer 2 changes, it updates, on L1, the latest proven L2 state. Alongside the proof, it publishes the state diff on Ethereum. The state diff is the difference between the previous and new states.
```
Comment (stoobie) [inline components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/on-chain-data.adoc:8]:
```suggestion
Starknet is a Validity Rollup, which means that after consolidating and proving a set of Layer 2 changes, it updates, on L1, the latest proven L2 state. Alongside the proof, it publishes the state diff on L1. The state diff is the difference between the previous and new states.
```

**Review by stoobie — APPROVED** [review]
LGTM

**Pull request merged**